net/mmfd/files/lib/gluon/upgrade/430-add-mmfd-interface

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/lua
+
+local uci = require('luci.model.uci').cursor()
+
+uci:delete('network', 'mmfd')
+uci:section('network', 'interface', 'mmfd', {
+	proto = 'static',
+	ifname = 'mmfd0',
+	ip6addr = 'fe80::1/64'
+}
+)
+uci:save('network')
```

Comment:
This file is Gluon-specific and must be move into the *gluon-mesh-babel* package. Also, it still uses the old luci.model package.